src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        String [] str = new String[0];
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileName));
+            StringBuilder stringBuilder = new StringBuilder();
+            String value = reader.readLine();
+            if (value == null) {
+                String [] resultWords = new String[0];
+                return resultWords;
+            }
```

Comment:
is this check effective? can we remove it?

Reply:
yes, fixed it.